src/vivarium_public_health/population/base_population.py

```diff
@@ -289,34 +290,41 @@ class AgeOutSimulants(Component):
     ##############
 
     @property
-    def columns_required(self) -> list[str]:
-        """A list of the columns this component requires that it did not create."""
-        return self._columns_required
+    def columns_created(self) -> list[str]:
+        return ["is_aged_out", "age_out_time"]
 
     #####################
     # Lifecycle methods #
     #####################
 
-    def __init__(self):
-        super().__init__()
-        self._columns_required = None
-
-    # noinspection PyAttributeOutsideInit
     def setup(self, builder: Builder) -> None:
         self.config = builder.configuration.population
-        if self.config.untracking_age is not None:
```

Comment:
What happens if this is None now?

Reply:
it's still used `on_timestep_cleanup` where if it's none it just returns and doens't age people out. But what we're doing here doesn't really have to do with untracking_age per se - we're just getting rid of `columns_required` (remember that's not a thing anymore)